README.md

```diff
@@ -16,7 +16,7 @@ Additional description about the project and its features.
 
 ## Live Demo
 
-[Live Demo Link](https://anselemodims.github.io/To-Do/dist/)
+[Live Demo Link](https://anselemodims.github.io/To-Do/src/)
 
 
 ## Getting Started
```

Comment:
- [OPTIONAL] It would be better to fix the broken MIT link in your README file. ♻️